test/help/main/expected-stdout

```diff
@@ -14,41 +14,41 @@ Usage:
   %s [command]
 
 Available Commands:
-  status         Print info about current directory.
+  status                         Show information about working directory.
 
-  sync
-    init         Init local project directory and perform the first pull.
-    diff         Print differences between local and remote state.
-    pull         Pull configurations to the project directory.
-    push         Push configurations to the Keboola project.
+  sync                           Synchronization between local directory and project.
+  sync init                      Initialize a new local directory and run "sync pull".
+  sync pull                      Sync project to the local directory.
+  sync push                      Sync local directory to the project.
+  sync diff                      Show differences between local directory and project.
 
-  local
-    validate     Validate the local project directory.
-    persist      Detect created/deleted configs/rows.
-    create
-      config     Create config.
-      row        Create config row.
-    encrypt      Encrypt all unencrypted secrets.
-    fix-paths    Normalize all local paths.
-    template
-      use        Apply template.
+  ci                             Manage CI/CD pipeline.
+  ci workflows                   Generate workflows for GitHub Actions.
 
-  ci
-    workflows    Generate Github Actions workflows.
+  local                          Operations performed in the local directory.
+  local create                   Create an object in the local directory.
+  local create config            Create an empty configuration.
+  local create row               Create an empty configuration row.
+  local persist                  Detect new/deleted configurations and rows.
+  local encrypt                  Encrypt all unencrypted secrets.
+  local validate                 Validate the local directory.
+  local fix-paths                Ensure that all paths match configured naming.
+  local template                 Manage templates in the local directory.
+  local template use             Apply template.
```

Comment:
Ukazalo sa, ze tomu predchadzajucemu `tree` sposobu zapisu pod-prikazov uzivatelia nerozumeju.
Tj. co maju napisat do konzoly, aby sa prikaz spustil.
Takze vypisujem prikazy cele  + aktualizoval som jednotlive hlasky.

Najlepsie asi bude, ked pozries na tento subor ako celok: https://github.com/keboola/keboola-as-code/blob/ddeae4531b8aff166027c55590dda1f14c673ab2/test/help/main/expected-stdout

Reply:
Aha, jasně. Takhle to vypadá taky ok. 👍